lib/email_parser.rb

```diff
@@ -16,6 +16,7 @@ def parse
     # within the regular expression, we are identifying two possible strings to split on:
     # ", " or " "
     # the pipe | represents OR in a regular expression 
+    # .uniq is an array method that removes duplicate values of self and returns a new array
```

Comment:
Thanks @Pete-Fowler ! I recommend one minor edit: replace "of self" with "from the array it is called on". If you could make that change I'll go ahead and merge in the PR.

Thanks for contributing to the Flatiron School curriculum 💙

Reply:
OK, I made the change and pushed to my fork of the solutions branch. Thanks!